.bazelrc

```diff
@@ -30,6 +30,12 @@ build --incompatible_enable_cc_toolchain_resolution
 # accessed like we do in util/BUILD
 build --workspace_status_command=util/get_workspace_status.sh
 
+# Enable more compiler warnings.
+build --copt=-Wunused-parameter
```

Comment:
@cfrantz any objections to these additions?

Reply:
This sort of compiler configuration really belongs in the compiler configuration repo:
https://github.com/lowRISC/crt

Reply:
Thanks for the pointer! I moved those settings to https://github.com/lowRISC/crt/pull/37